### DISCUSSION TITLE
How "MaxClients 1" works?

### DISCUSSION THREAD

**Discussion by CCCAUCHY** [tinyproxy/tinyproxy#541] [Q&A]
Does "MaxClients 1" means tinyproxy only serves the latest client?

**Comment by rofl0r** [comment]
it means only one connection will be accepted, the rest rejected. that would typically be the first connection. as soon as the web-page is served with Connection: close or timeout is hit it will be available for the next client. note that modern browsers usually start at least 10 or so parallel connections to fetch the different resources. making only 1 connection available will either break the majority of websites, or make your browsing really slow.
